### ISSUE TITLE
Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by M4iku-LandscapeCL** [nullstack/nullstack#79]
Hello, i try to run the base project but i get this error:
```
C:\phagane\node_modules\nullstack\scripts\index.js:30
    const [file, loader, ...trace]  = stats.toJson('errors-only', {colors: true}).children[0].errors[0].split('\n');
                                                                                                       ^
TypeError: Cannot read property 'split' of undefined
    at logTrace (C:\phagane\node_modules\nullstack\scripts\index.js:30:104)
    at C:\phagane\node_modules\nullstack\scripts\index.js:65:5
    at Watching.handler (C:\phagane\node_modules\webpack\lib\MultiCompiler.js:225:10)
    at C:\phagane\node_modules\webpack\lib\Watching.js:99:9
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\phagane\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:15:1)
    at AsyncSeriesHook.lazyCompileHook (C:\phagane\node_modules\tapable\lib\Hook.js:154:20)
    at Watching._done (C:\phagane\node_modules\webpack\lib\Watching.js:98:28)
    at C:\phagane\node_modules\webpack\lib\Watching.js:73:19
    at Compiler.emitRecords (C:\phagane\node_modules\webpack\lib\Compiler.js:499:39)
    at C:\phagane\node_modules\webpack\lib\Watching.js:54:20
```

Node -V : 14.15.5

**Comment by Mortaro** [comment]
Heyo, thanks for reporting, is this a clean repo with create-nullstack-app? can you try with version 0.11.0 pls (atm its ~0.11.0 which resolves to 0.11.1)?

**Comment by Mortaro** [comment]
Seems to be fixed in 0.12.1 please reopen if you experience this again!
